store/tikv/prewrite.go

```diff
@@ -128,6 +128,11 @@ func (action actionPrewrite) handleSingleBatch(c *twoPhaseCommitter, bo *Backoff
 					c.run(c, nil)
 				}
 			}
+			c.mu.Lock()
+			if prewriteResp.MinCommitTs > c.minCommitTS {
+				c.minCommitTS = prewriteResp.MinCommitTs
```

Comment:
And if `MinCommitTS` is 0, it means async commit cannot proceed due to some reason. We should also set `useAysncCommit` to false.